roles/advanced_dhcp_server/templates/dhcpd.subnet.conf.j2

```diff
@@ -107,29 +107,33 @@
 {% for host in range %}
   {% set current_hostvars = hostvars[host] %}{# buffer #}
   {% if current_hostvars['network_interfaces'] is defined and current_hostvars['network_interfaces'] is iterable %}
-    {#- Define iPXE rom to use #}
-    {% if current_hostvars['ep_ipxe_driver'] == 'snponly' %} {# Select driver first #}
-      {% set ipxe_driver = 'snponly_' %}
-    {% elif current_hostvars['ep_ipxe_driver'] == 'snp' %}
-      {% set ipxe_driver = 'snp_' %}
-    {% else %}{# Assume everything else is default driver #}
-      {% set ipxe_driver = '' %}
-    {% endif %}
-    {% if current_hostvars['ep_hardware']['cpu']['architecture'] == 'arm64' %} {# Now select architecture #}
```

Comment:
Hello guys, please remove the space between the end of the if conditions and the beginning of comments.
That space will broke the configuration file and the dhcpd service will not start.